foreshadow/steps/cleaner.py

```diff
@@ -51,13 +53,39 @@ def fit_transform(self, X, *args, **kwargs):
         Returns:
             A transformed dataframe.
 
+        """
+        Xt = super().fit_transform(X, *args, **kwargs)
+        self._check_empty_dataframe(Xt)
```

Comment:
Do we still need override fit_transform? Shouldn’t it be default calling the transform method and already have these logic?

Reply:
We actually need it, at least for now since we have an open issue of `fit` method calling `fit_transform` method.